RecoDataProducts/inc/CrvRecoPulseFlags.hh

```diff
@@ -0,0 +1,14 @@
+#ifndef RecoDataProducts_CrvRecoPulseFlags_hh
+#define RecoDataProducts_CrvRecoPulseFlags_hh
+
+#include <bitset>
+
+namespace mu2e 
+{
+  enum CrvRecoPulseFlagEnums{failedFit=0, unused=1, unused2=2};
```

Comment:
How do we plan on using this?
How do you define failedFit?
What's unused and unused2?

Reply:
I'm not using it yet. But I can imagine that we may want to use it later at the coincidence finder.
The unused flags are place holders for additional flags which I may add in the future. I can remove them.

Reply:
Fits are considered "failed fits", if the ROOT fit did not succeed, or if one or more fit parameters are at their limits.